client/Pages/README.md

```diff
@@ -0,0 +1,103 @@
+# Pages
+
+This module defines and exports metadata which describes the pages in the Strimzi UI, and how they relate to the [`Panel`](../Panels/README.md) components that implement them. This metadata is then processed by the UI's [navigation logic](../Bootstrap/Navigation) at runtime to determine what pages are shown and available to a user, but also how they are rendered in context to the wider UI and relate to one another. The motivation for this is three fold: to separate the components that implement a page and navigation (to avoid cross contamination), to enable extensibility for new pages in future, and allow re use of the same `Panel` components in numerous contexts with minimal changes.
+
+## Page Metadata Schema
+
+The schema used to define pages in the Strimzi-ui is designed around capability, and the scenarios those capabilities are presented in. The exported object should be named to reflect the capability it provides, and the metadata contained to specify the `Panel` component used to provide it (`contentComponent`), and where (and how) it should be presented (`contexts`). The below example (pseudo) shape shows the structure expected:
+
+```
```

Comment:
Is this a duplication of the content above?

Reply:
By above, do you mean in the Pages README? If so, I see the text above this section as more of a summary, and this as more detail about the structure itself.